syft/pkg/cataloger/dotnet/libman_json.go

```diff
@@ -0,0 +1,107 @@
+package dotnet
+
+import (
+	"encoding/json"
+	"path"
+	"strings"
+
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/internal"
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+type libmanJSON struct {
```

Comment:
Found this as the current reference updated as recently as 5 hours ago. Might be a good inclusion as a comment in case we see any breakage in the future or if the version upgrades.
https://github.com/aspnet/LibraryManager/wiki/libman.json-reference